specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -492,6 +500,155 @@
           }
         }
       }
+    },
+    "/weather/airQuality/current/{format}": {
+      "get": {
+        "description": "**Get Current Air Quality**\n\n**Applies to**: S1 and G2 pricing tiers.\n\nThe service provides the detailed information about the concentration of pollutant and overall status for current air quality.",
+        "operationId": "Weather_GetCurrentAirQuality",
+        "x-ms-examples": {
+          "CurrentAirQuality": {
+            "$ref": "./examples/GetCurrentAirQuality.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/Language"
+          },
+          {
+            "name": "query",
```

Comment:
consider a refined non-string version as in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Weather/preview/1.0/weather.json#L99-L114

Reply:
Updated